extensions/amp-google-read-aloud-player/0.1/test/test-amp-google-read-aloud-player.js

```diff
@@ -0,0 +1,234 @@
+import '../amp-google-read-aloud-player';
+
+import {expect} from 'chai';
+
+import {createElementWithAttributes} from '#core/dom';
+import {Layout_Enum} from '#core/dom/layout';
+
+import {Services} from '#service';
+
+import {macroTask} from '#testing/helpers';
+
+const IFRAME_BASE_URL =
+  'https://www.gstatic.com/readaloud/player/web/api/iframe/index.html';
+
+const API_KEY = 'YOUR_API_KEY';
+const TRACKING_IDS = 'YOUR_TRACKING_IDS';
+const URL = 'http://www.example.com';
+const CALL_TO_ACTION_LABEL = 'Listen to this article';
+const LOCALE = 'en';
+const INTRO = 'http://www.example.com/intro.mp3';
+const OUTRO = 'http://www.example.com/outro.mp3';
+const AD_TAG_URL = 'http://www.example.com/ad';
+const VOICE = 'SOME_VOICE';
+
+const LAYOUT_ATTRS = {
+  layout: Layout_Enum.FIXED_HEIGHT,
```

Comment:
Nit: `fixed-height` is inferred by the runtime from only setting `height`, you don't need this attribute.
```suggestion
```